keyboards/plexus75/config.h

```diff
@@ -23,7 +23,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRODUCT_ID      0x5338
 #define DEVICE_VER      0x0001
 #define MANUFACTURER    nasp
-#define PRODUCT         Plexus75
+#define PRODUCT         nasp Plexus75
```

Comment:
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs
> don’t repeat MANUFACTURER in the PRODUCT value


```suggestion
#define PRODUCT         Plexus75
```

Reply:
ok